php83/upstream.conf

```diff
@@ -0,0 +1,4 @@
+# Upstream to abstract backend connection(s) for PHP.
+upstream php82 {
```

Comment:
Hi, shouldn't this be "php83"? Provisioning fails when trying to include 8.2 and 8.3. 
This is the error I get:

`Jan 19 09:36:58 vvv nginx[16679]: nginx: [emerg] duplicate upstream "php82" in /etc/nginx/upstreams/php83.conf:2 ` 

I can correct this manually and restart Nginx without issue, but reprovisioning overwrites it and throws the error again.

Reply:
Ok PR approved, you can do a new provision and everything will be fixed